src/pages/index.js

```diff
@@ -0,0 +1,7 @@
+import SignIn from "./Auth/Signin";
+import SignUp from "./Auth/Signup";
+
+import Home from "./Home/Home";
+import VideoListing from "./VideoListing/VideoListing";
+
+export { Home, VideoListing, SignIn, SignUp };
```

Comment:
you are importing all files in index.js and export it from here , this is good approach ,I liked it